specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2618 @@
+          "description": "Resource ID of the amlFilesystem.",
+          "readOnly": true,
+          "$ref": "#/definitions/URLString"
+        },
+        "location": {
+          "description": "Region name string.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of amlFilesystem.",
+          "readOnly": true,
+          "$ref": "#/definitions/amlFilesystemName"
+        },
+        "type": {
+          "description": "Type of the filesystem; Microsoft.StorageCache/amlFilesystem",
+          "readOnly": true,
+          "type": "string"
+        },
+        "identity": {
+          "$ref": "#/definitions/IdentityProperties",
+          "description": "The managed identity used by the amlFilesystem, if configured."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of the amlFilesystem.",
+          "properties": {
+            "storageCapacityTiB": {
+              "description": "The size of this filesystem, in TiB.  This may be rounded up.",
+              "type": "number",
+              "format": "float",
+              "readOnly": true
+            },
+            "health": {
+              "description": "Health of the amlFilesystem.",
+              "readOnly": true,
+              "$ref": "#/definitions/amlFilesystemHealth"
+            },
+            "zones": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "description": "Availability zones for nodes.",
+              "readOnly": true
+            },
+            "provisioningState": {
+              "description": "ARM provisioning state, see https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#provisioningstate-property",
+              "enum": [
+                "Succeeded",
+                "Failed",
+                "Cancelled",
+                "Creating",
+                "Deleting",
+                "Updating"
+              ],
+              "x-ms-enum": {
+                "name": "ProvisioningStateType",
+                "modelAsString": true
+              },
+              "type": "string"
+            },
+            "subnet": {
+              "description": "Subnet used exclusively for the amlFilesystem.",
+              "$ref": "#/definitions/URLString"
+            },
+            "MGSAddress" : {
+              "description": "IPv4 Address",
+              "readOnly" : true,
+              "type" : "string"
+            },
+            "mountCommand" : {
+              "description" : "Recommended command to mount the Lustre filesystem",
+              "readOnly" : true,
+              "type" : "string"
+            },
+            "lustreVersion" : {
+              "type" : "string",
+              "description" : "The version of Lustre running in the amlFilesystem",
+              "readOnly": true
+            },
+            "throughputCapacity" : {
```

Comment:
In storage, Capacity typically refers to the amount of data that can be stored. I suggest something like throughputProvisioned or something else that avoids overloading terms like capacity.

Reply:
Fixed.  Examples updated to match using throughputProvisioned